c_src/Makefile

```diff
@@ -110,8 +110,11 @@ $(C_SRC_OUTPUT): build_erl_jq
 	cp $(JQSRC_DIR)/.libs/$(LIBJQ) $(PRIV_DIR)/
 	cp $(JQSRC_DIR)/modules/oniguruma/src/.libs/$(LIBONIG) $(PRIV_DIR)/
 	install_name_tool -change $(LIBJQ_PREFIX)/lib/$(LIBONIG) "@loader_path/$(LIBONIG)" $(PRIV_DIR)/$(LIBJQ)
+	install_name_tool -change $(LIBJQ_PREFIX)/lib/$(LIBONIG) "@loader_path/$(LIBONIG)" $(PRIV_DIR)/$(LIBJQ)
```

Comment:
I wonder about this step. When this library is included as a dependency to EMQX, and the user installs EMQX that has been built on a different machine, the `install_name_tool` command is not executed on the host machine. Is this a problem?

Reply:
Ok, thanks for the explanation.